ChangeLog.md

```diff
@@ -5,6 +5,8 @@
 - Oav runner support specifying option 'runId' with option 'from' and/or 'to' to debug.
 - Oav runner support using env variable 'TEST_SCENARIO_JSON_ENV' to override variables in env.json.
 - Replace Oav runner option 'cleanUp' with 'skipCleanUp'.
+- Update README.md add gif to show API test
```

Comment:
Why not bump a new version for these new changes?

Reply:
2.7.0 is not released yet.